docs/deployment.md

```diff
@@ -55,3 +55,32 @@ Folder | URL prefix
 
 To add more folders, follow [Cloudinary's instructions](https://cloudinary.com/documentation/fetch_remote_images#auto_upload_remote_resources).
 
+## Django Migrations: is my deploy safe?
+
+### Context
+
+[Preboot](https://devcenter.heroku.com/articles/preboot) is a feature from Heroku that prevents downtime when deploying: Heroku still serves traffic to old dynos while creating new dynos at the same time. For up to 3 minutes, two versions of the code are running and talking to the same DB: backward-incompatible changes to the DB result in internal server errors from the old code.
+
+### How to apply backward-incompatible migrations
+
+Always announce in `#mofo-engineering` when you deploy backward-incompatible changes.
```

Comment:
Do we want to change this to something slightly stronger, like "Announce your intention to deploy in `#mofo-engineering`, and hold off deploying until people have indicated they've seen your announcement. If there is no one around, do not deploy."?

Reply:
done!